src/Microsoft.AspNet.Server.Kestrel/KestrelEngine.cs

```diff
@@ -1,26 +1,31 @@
 // Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using Microsoft.AspNet.Server.Kestrel.Networking;
-using System.Collections.Generic;
-using System.Threading.Tasks;
 using Microsoft.AspNet.Server.Kestrel.Http;
 using Microsoft.Framework.Runtime;
+using Microsoft.Framework.Logging;
+using Microsoft.Framework.Logging.Console;
+using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Threading.Tasks;
 
 namespace Microsoft.AspNet.Server.Kestrel
 {
     public class KestrelEngine : IDisposable
     {
 
-        public KestrelEngine(ILibraryManager libraryManager)
+        public KestrelEngine(ILibraryManager libraryManager, ILoggerFactory loggerFactory)
         {
             Threads = new List<KestrelThread>();
             Listeners = new List<Listener>();
             Memory = new MemoryPool();
             Libuv = new Libuv();
 
+            loggerFactory.AddConsole();
```

Comment:
Sure, but when I think about it now, I think it makes sense for Kestrel to always write to the console (not through a logger). The usability of Kestrel is otherwise fairly difficult especially to a beginner.